mlflow/tracking/client.py

```diff
@@ -2667,6 +2671,26 @@ def create_model_version(
             await_creation_for=await_creation_for,
         )
 
+    def copy_model_version(self, src_model_uri, dest_name) -> ModelVersion:
+        """
+        Copy a model version from one registered model to another as a new model version.
+
+        :param src_model_uri: the model URI of the model version to copy
```

Comment:
```suggestion
        :param src_model_uri: the model URI of the model version to copy (e.g., `"models:/foo/bar"`)
```

Can we add an example value?

Reply:
Sounds good!